src/components/StreamingMedia/createTrackMediaSession.js

```diff
@@ -10,6 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 import PlaybackState from "./constants/playbackState.js";
+import {isEmptyObject} from "../../utils/index.js";
```

Comment:
Don't forget to remove this unused import

Reply:
good catch, thank you